src/client/Microsoft.Identity.Client/Platforms/Android/EmbeddedWebview/AuthenticationAgentActivity.cs

```diff
@@ -26,25 +25,27 @@ protected override void OnCreate(Bundle bundle)
             base.OnCreate(bundle);
             // Create your application here
 
-            WebView webView = new WebView(ApplicationContext);
-            var relativeLayout = new RelativeLayout(ApplicationContext);
-            webView.LayoutParameters = new RelativeLayout.LayoutParams(RelativeLayout.LayoutParams.MatchParent, RelativeLayout.LayoutParams.MatchParent);
+            WebView webView = new WebView(this);
+            var relativeLayout = new RelativeLayout(this);
+            webView.LayoutParameters = new RelativeLayout.LayoutParams(global::Android.Views.ViewGroup.LayoutParams.MatchParent, global::Android.Views.ViewGroup.LayoutParams.MatchParent);
 
             relativeLayout.AddView(webView);
             SetContentView(relativeLayout);
 
             string url = Intent.GetStringExtra("Url");
+
```

Comment:
Do you want to keep this comment? and the followings?

Reply:
i did for us for debugging, but have removed. 

---
In reply to: [353459703](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1531#discussion_r353459703) [](ancestors = 353459703)